configuration/pih/statusData/hiv/nextMedicationPickupDate.sql

```diff
@@ -0,0 +1,27 @@
+# Retrieve the  Next Medication Pickup Date (PIH:5096) from the last HIV drug dispensing encounter.
+#   If this date is more than 28 days ago, highlight it in red.
+#   If there have been no HIV Drug Dispensing encounters for this patient, display “No medication pickups” in red.
```

Comment:
Nit, but shouldn't these comments be in the hivStatuses.yml and the comment here should just be about what this query returns (number of med pickup encounters and most recent med pickup encounter, I believe)?

Reply:
Probably.  These are something I initially copied in here from the ticket in order to define what I was doing when I was writing the query.